src/fides/api/db/system.py

```diff
@@ -137,7 +137,7 @@ async def upsert_privacy_declarations(
 
 async def upsert_cookies(
     async_session: AsyncSession,
-    cookies: Optional[List[Dict]],  # CookieSchema
+    cookies: List[Dict],  # CookieSchema
```

Comment:
There might be some places in the codebase where we're accessing cookies and if they're not there, returning None as a default - can you check that, and make it an empty list if needed?

Reply:
If this is supposed to be a cookie schema, why isn't it typed as such? I want to make sure I don't undo anything that has a specific reason

Reply:
> There might be some places in the codebase where we're accessing cookies and if they're not there, returning None as a default - can you check that, and make it an empty list if needed?

Is just related to how below you've stopped doing "cookies or []" so I just want to make sure we always have something to iterate through.

Reply:
understood! For the sake of type cleanliness, I'll do an explicit conversion here